File: Models/DomainModels/feedback.cs
namespace LenzPerson.api.Models.DomainModels
{
    public class feedback
    {
        public int Id { get; set; }
        public int BId { get; set; }
        public int PId { get; set; }
        public int CId { get; set; }
        public string CName { get; set; }
        public string Comment  { get; set; }
    }
}

File: src/app/models/feedback.ts
export class feedback{
    PId : number;
    BId : number;
    CId: number;
    CName: string 
    Comment: string;

    


    constructor(
        PId: number,
        BId: number,
        CId: number,
        CName: string,
        Comment: string,
    ) {
      this.PId = PId;
      this.BId = BId;
      this.CId = CId;
      this.CName = CName;
      this.Comment = Comment;
      
    }



}